cli/main.rs

```diff
@@ -314,6 +314,7 @@ fn run_script(flags: DenoFlags, argv: Vec<String>) {
     worker
       .execute_mod_async(&main_module, false)
       .and_then(move |()| {
+        js_check(worker.execute("denoMain.triggerLoadEvent()"));
```

Comment:
I feel we might want to give it a more general name (I am thinking that we might add more lifecycle hooks at some point)

Reply:
Changed to `denoMain.lifecycles.onLoad` for now. Maybe we can add more lifecycles in this namespace. I'm open to any further suggestion!